src/test/regress/expected/multi_complex_expressions.out

```diff
@@ -224,7 +224,7 @@ SELECT count(*) FROM lineitem
 	WHERE 0 != 0;
  count 
 -------
-     0
+      
```

Comment:
It'd be quite good to mention that a) in the commit message, b) in the test file itself.